daprdocs/content/en/reference/arguments-annotations-overview.md

```diff
@@ -38,6 +38,7 @@ This table is meant to help users understand the equivalent options for running
 | `--app-protocol` | `--app-protocol` | `-P` | `dapr.io/app-protocol` | Tells Dapr which protocol your application is using. Valid options are `http` and `grpc`. Default is `http` |
 | `--sentry-address` | `--sentry-address` | | not supported | Address for the Sentry CA service |
 | `--version` | `--version` | `-v` | not supported | Prints the runtime version |
+| `--dapr-graceful-shutdown-seconds` | not supported | | `dapr.io/graceful-shutdown-seconds` | Graceful shutdown duration in seconds for dapr, the maximum duration before force shutdown when waitting all in-progress requests to complete. Defaults to `5`, and maximum to `30` which is equivalent to Kubernetes termination grace period. Any vaule larger than the minimum duration will be setting to the minimum duration.  |
```

Comment:
Additionally, please re-evaluate this sentence as I think you might either mean "any value larger than the maximum" or "any value smaller than the minimum"?
`Any vaule larger than the minimum duration will be setting to the minimum duration.`

Reply:
Yes, thank you for pointing it out. What I really want to describe is`any value larger than the maximum`.
done.

Reply:
@Taction  - Can you accept the suggested changes from @willtsai

Reply:
I have accepted these suggestions in fact, but the code has a minor change, so this doc has been changed too. 
I'll fix dco asap.

Reply:
The limit of `maximum duration` has been removed after a discussion about it.